src/client/components/CreateSeason.jsx

```diff
@@ -10,38 +10,36 @@ import { TeamForm } from './seasonform/TeamForm';
 import { ParticipantForm } from './seasonform/ParticipantForm';
 import { MappingForm } from './seasonform/MappingForm';
 
-import './styling/create-season.css'
+import './styling/create-season.css';
 
 export const CreateSeason = () => {
     const { state } = useContext(StoreContext);
-    const [token, setToken] = useState()
-    const navigate = useNavigate();
 
-    console.log(state)
+    const navigate = useNavigate();
 
-    const competitionId = state.competitions[0].id;
+    const competitionId = state.selectedCompetition[0].id;
```

Comment:
is `selectedCompetition` still an array containing an object?

Reply:
Good shout, Dr. B!